ddtrace/appsec/internal/management.py

```diff
@@ -0,0 +1,67 @@
+"""
+The AppSec module detects security anomalies from inside the application.
+
+Test it with::
+
+    DD_APPSEC_ENABLED=true FLASK_APP=hello.py ddtrace-run flask run
+
+"""
+import logging
+import os.path
+from typing import Any
+from typing import List
+from typing import Mapping
+
+import attr
+
+from ddtrace import Span
+from ddtrace.appsec.internal.protections import BaseProtection
+from ddtrace.utils.formats import get_env
+
+
+log = logging.getLogger(__name__)
+
+
+@attr.s(eq=False)
+class Management(object):
+    """
+    AppSec module management.
+    """
+
+    protections = attr.ib(type=List[BaseProtection], default=[])
```

Comment:
also, should `protections` be private? I suppose it isn't exposed because we don't expose the management class anywhere in the public API but I also don't see a reason it should be public on the class.